modules/common/src/main/PackageObject.scala

```diff
@@ -190,8 +190,11 @@ trait WithPlay { self: PackageObject =>
 
   implicit final class LilaPimpedFiniteDuration(self: FiniteDuration) {
 
-    def toTenths: Long = self.toMillis / 100
+    def roundSeconds = math.round(self.toMillis.toDouble / 1000)
+    def roundTenths = math.round(self.toMillis.toDouble / 100)
```

Comment:
I fail to see the difference between `self.toMillis / 100` and `math.round(self.toMillis.toDouble / 100)`. They should return the same `Long`. Do they not?